Dockerfile

```diff
@@ -16,7 +16,7 @@ ENV CONFIG_PATH="/opt/janus/etc/janus"
 # docker build arguments
 ARG BUILD_SRC="/usr/local/src"
 
-ARG JANUS_VERSION="v0.10.3"
+ARG JANUS_VERSION="v1.0.0"
 ARG JANUS_LIBNICE_VERSION="0.1.17"
```

Comment:
@jhon-freitez-jemmic Also update the other dependencies.
For example libnice here is still version 0.1.17, while on https://github.com/meetecho/janus-gateway at least 0.1.18 is recommended.
Also check if any other changes in dependencies are needed (e.g. new dependencies).